src/nvim/eval.c

```diff
@@ -8132,6 +8132,8 @@ static void f_execute(typval_T *argvars, typval_T *rettv, FunPtr fptr)
     return;
   }
 
+  int save_msg_col = msg_col;
+  int should_restore_col = 0;
```

Comment:
use `bool` `true` and `false`

Reply:
Thanks for pointing out that. I almost forget c has bool. 

Since the PR has been created, what should I do now? 

1. Cancel and create a new one?
2. Rely on the one to merge it to update it?

Reply:
just update the PR. force-pushing is okay.